### PR TITLE
[CU-1zhbq2m] Develop STE "FAQ" section

### PR DESCRIPTION
**Pull request by vanessavmac**
## Changes
- Complete STE FAQ section skeleton 
- Update to include the Hi-fi designs
- Add SustainableTech folder to static/copy with JSON file for FAQ content
- Add src/static/ste/icons folder for clouds and geese images
- Deleted the `node_modules` directory, `package-lock.json` or `yarn.lock` files in the outer `waterloop-website-w20` 
- Added final business team content

Note: Sponsors and FAQ section share the same gradient background so once we combine the sections will need to coordinate the background color.

Screen Shot 2022-03-02 at 10 54 54 PM

## Type of change
- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?
- [x] Tested responsiveness in mobile and tablet mode
- [x] Successfully updated FAQ content using the JSON file 
- [x] Visited each waterloop website page to ensure the deleted `node_modules` did not cause any issues

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by jennchenn** [comment]
Task linked: [CU-1zhbq2m Develop STE "FAQ" section](https://app.clickup.com/t/1zhbq2m)

**Review by eevanwong — CHANGES_REQUESTED** [review]
image

Did you install dependencies in the main directory instead of the inside the waterloop-site folder?

**Comment by eevanwong** [comment]
Like the feedback I gave in your other PR, please remove the requested files in the main directory of `waterloop-website-w20`.

**Comment by vanessavmac** [comment]
Just made the changes! Everything is working as it should too. Should I also delete the yarn.lock file? I only deleted the node_modules, package-lock.json, and package-lock in the main directory.

**Comment by eevanwong** [comment]
Yep, please delete the yarn.lock as well, we only really care about the readme and the actual waterloop-site folder.

**Comment by vanessavmac** [comment]
Ok sounds good, I deleted the files.

**Comment by eevanwong** [comment]
Ok, looks good! However, I don't believe we have the content for these components yet. We'll hold off on merging it until then.

**Comment by vanessavmac** [comment]
Sounds good, thank you for reviewing + helping me make edits!

File: waterloop-site/package.json
Comment (williamhpark) [inline waterloop-site/package.json:9]: Instead of installing this new dependency, let's instead use the library we currently have already installed, `@material-ui/icons`.
To import the icon, you should be able to do something like:
`import ArrowDropDownIcon from '@material-ui/icons/ArrowDropDown';`

Note: We want to try to not install new dependencies unless there's a definite need for it that cannot be fulfilled by the currently installed ones.

File: waterloop-site/src/static/copy/SustainableTech/FAQ.json
Comment (williamhpark) [inline waterloop-site/src/static/copy/SustainableTech/FAQ.json:1]: Thank you for embedding the data here!

File: waterloop-site/src/components/SustainableTech/FAQ/collapsible.tsx
Comment (williamhpark) [inline waterloop-site/src/components/SustainableTech/FAQ/collapsible.tsx:44]: Could you rename this file and component to something a bit more specific, maybe `AccordionDropdown`? Also, please make the file name capitalized :)

File: waterloop-site/src/components/SustainableTech/FAQ/collapsible.tsx
Comment (williamhpark) [inline waterloop-site/src/components/SustainableTech/FAQ/collapsible.tsx:53]: The formatting looks off here, can you reformat the files with Prettier and commit again?

File: waterloop-site/src/components/SustainableTech/FAQ/collapsible.tsx
Comment (williamhpark) [inline waterloop-site/src/components/SustainableTech/FAQ/collapsible.tsx:3]: Rather than have an up arrow and down arrow as individual components, would be able to have just the one down arrow component and when the user expands the dropdown, the arrow flips 180 degrees (with a transition to make the rotation look smooth)?
Although this is a small detail, it would be awesome if you could do it.

File: waterloop-site/src/components/SustainableTech/FAQ/FAQ.tsx
Comment (williamhpark) [inline waterloop-site/src/components/SustainableTech/FAQ/FAQ.tsx:2]: We generally try to separate dependency imports and custom imports at the top of the file by a newline. Can you move the `Collapsible` and `Content` (custom) imports after the `React` and `styled` (dependency) imports, and separate the two groups by a newline?

File: waterloop-site/src/components/SustainableTech/FAQ/FAQ.tsx
Comment (williamhpark) [inline waterloop-site/src/components/SustainableTech/FAQ/FAQ.tsx:21]: Formatting here looks a bit off as well, needs to be reformatted with Prettier.

File: waterloop-site/src/components/SustainableTech/FAQ/collapsible.tsx
Comment (williamhpark) [inline waterloop-site/src/components/SustainableTech/FAQ/collapsible.tsx:38]: For now this is no problem, but just a note that we'll probably want to move the colour theme to a separate file later, so other STE pages to access as well, instead of hard-coding it like it currently is.

File: waterloop-site/src/components/SustainableTech/FAQ/collapsible.tsx
Comment (williamhpark) [inline waterloop-site/src/components/SustainableTech/FAQ/collapsible.tsx:5]: Can you name this just `Props`, so it's a bit cleaner?

**Review by williamhpark — CHANGES_REQUESTED** [review]
Awesome start! I have a few changes I would like you to make, let mw know if you want clarification and I'd be happy to call and talk through anything.
Also, please commit any new changes as incrementally as you can (e.g. Prettier reformatting in its own commit, etc.)

**Comment by williamhpark** [comment]
IMPORTANT!
So as @eevanwong stated we don't actually use/need the `node_modules` directory, `package-lock.json` or `yarn.lock` files in the outer `waterloop-website-w20`... those were committed by accident a few terms back and we should remove them from the repo. However, for each PR we want to make sure that we're only addressing code changes for the specific ticket, so I would recommend reverting back to how those were before. We can address removing those files and testing for regressions in a different ticket/PR.
Also more importantly, we DO need the `yarn.lock` file in the inner `waterloop-site` directory. I noticed you deleted the `yarn.lock` file from the `waterloop-site` - please revert that change back because we need that to be committed in our source control. Refer to this link if you want more detail: https://classic.yarnpkg.com/en/docs/migrating-from-npm.

File: waterloop-site/src/components/SustainableTech/FAQ/collapsible.tsx
Comment (williamhpark) [inline waterloop-site/src/components/SustainableTech/FAQ/collapsible.tsx:56]: I tried out your changes on my local and it looks great, but when the panel expands, would you be able to make the expanding smoother? This would have to be done using CSS transitions, refer here for more info: https://www.w3schools.com/css/css3_transitions.asp

File: waterloop-site/src/components/SustainableTech/FAQ/collapsible.tsx
Comment (williamhpark) [inline waterloop-site/src/components/SustainableTech/FAQ/collapsible.tsx:6]: BTW how come you made this optional? Shouldn't the dropdown always just be either open or not open?

File: waterloop-site/src/components/SustainableTech/FAQ/FAQ.tsx
Comment (williamhpark) [inline waterloop-site/src/components/SustainableTech/FAQ/FAQ.tsx:42]: Make sure that there's one space after each colon in the CSS

File: waterloop-site/src/components/SustainableTech/FAQ/AccordionDropdown.tsx
Comment (williamhpark) [inline waterloop-site/src/components/SustainableTech/FAQ/AccordionDropdown.tsx:45]: 0.8s is kind of long. Could you make this a bit shorter, maybe 0.5?

File: waterloop-site/src/components/SustainableTech/FAQ/AccordionDropdown.tsx
Comment (williamhpark) [inline waterloop-site/src/components/SustainableTech/FAQ/AccordionDropdown.tsx:66]: Could you shorten this transition time too? (0.5)

**Review by williamhpark — CHANGES_REQUESTED** [review]
Looks great! Just some small additional suggestions :)

File: waterloop-site/src/static/ste/icons/cloud.svg
Comment (williamhpark) [inline waterloop-site/src/static/ste/icons/cloud.svg:1]: Can you pass this SVG markup, as well as the other ones, through [SVGOMG](https://jakearchibald.github.io/svgomg/) to clean it up a bit and also prettify?

Comment (williamhpark) [reply waterloop-site/src/static/ste/icons/cloud.svg:1]: Also, could you move all SVGs too `waterloop-website-w20/waterloop-site/src/static/img/sustainable-tech`?

File: waterloop-site/src/static/copy/SustainableTech/FAQ.json
Comment (williamhpark) [inline waterloop-site/src/static/copy/SustainableTech/FAQ.json:11]: The speaker's name is "Mike Morrice"... can you include it?

File: waterloop-site/src/static/copy/SustainableTech/FAQ.json
Comment (williamhpark) [inline waterloop-site/src/static/copy/SustainableTech/FAQ.json:15]: Speaker name here as well.

File: waterloop-site/src/static/copy/SustainableTech/FAQ.json
Comment (williamhpark) [inline waterloop-site/src/static/copy/SustainableTech/FAQ.json:16]: I like the way you formatted this answer (it's better than what's currently shown in the Google doc), but I feel like using the word "speaking" sounds better than "speak" at the end of this sentence, do you agree?

**Review by williamhpark — APPROVED** [review]
Looks great, besides some more additional veryy minor changes. Once they're addressed, I think it looks good to merge (so I'm approving now).

**Review by eevanwong — APPROVED** [review]
LGTM!

Awesome job Vanessa! Thanks for keeping up with all the feedback :).

**Pull request merged**
